src/main/scala/beam/router/TravelTimeObserved.scala

```diff
@@ -88,14 +89,15 @@ class TravelTimeObserved @Inject()(
             uniqueTimeBins
               .foreach { timeBin =>
                 val key = PathCache(origin.tazId, destination.tazId, timeBin)
+                counts(key) += 1
```

Comment:
This doesn't seem right. I would expect to build the full count and then write it. This way it adds and immediately outputs the current state. It will always go 1 then 2 then 3 until the final count for the key afaict.

Reply:
@JustinPihony thanks a lot. Stupid mistake. Fixing.